File: day 023/homework/homework04.py
# 4) შექმენით ფუნქცია რომელსაც გადაეცემა ლისტი.
# ფუნქციამ უნდა იპოვოს ლისტში უდიდესი რიცხვი.
def longest_num(list):
    num = 0
    for i in list:
        if i > num:
            num = i
    return num
print(longest_num([3,5,8,2,9,1,7,8,5,3]))

File: day 086/homework/homework04.js
function func(num){
    if (num < 0 ){
        return 'This number is negative'
    } else if (num === 0){
        return 'This number is 0'
    } else{
        return 'This number is positive'
    }
}
console.log(func(Number(prompt('Enter number: '))))